checks/apps/lammps/lammps_check.py

```diff
@@ -9,12 +9,21 @@
 import reframe.utility.sanity as sn
 
 
+class fetch_lampps_resources(rfm.RunOnlyRegressionTest):
+    executable = 'wget https://jfrog.svc.cscs.ch/artifactory/cscs-reframe-tests/LAMMPS.tar.gz && tar -xvf LAMMPS.tar.gz'
```

Comment:
```suggestion
        jfrog = 'https://jfrog.svc.cscs.ch/artifactory/cscs-reframe-tests'
        executable = f'wget {jfrog}/LAMMPS.tar.gz && tar -xvf LAMMPS.tar.gz'
```
+ maybe jfrog should be set in the config file(s) ?